### ISSUE TITLE
External tools, wrappers etc

### ISSUE THREAD

**Issue by danyaljj** [IllinoisCogComp/illinois-cogcomp-nlp#174]
Completing this list over time ... maybe we should have a place for them in the readme file. 

https://github.com/ILCogCompCuratorToHadoop/CuratorHadoopInterface

https://github.com/cwhits/cogcomp

https://github.com/dkpro/dkpro-core/tree/ba33629fc0f077337f9af39e38e1b58531e1674e/dkpro-core-lbj-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/lbj

**Comment by cogcomp-dev** [comment]
What exactly do you want to do here? Just add a list of references to projects that use our tools? Where do you think this should go?

**Comment by danyaljj** [comment]
> Just add a list of references to projects that use our tools?

Not exactly. What I meant was external *utility* projects, like someone who has python wrapper etc.

**Comment by danyaljj** [comment]
C# sentence splitter: 
https://github.com/saurabhj/SentenceSplitter.Net
